### ISSUE TITLE
Multi-tenant

### ISSUE THREAD

**Issue by kkratterf** [tokens-bruecke/figma-plugin#6]
Is there a possibility in the future to have a multitenant version?
at the moment I have 4 design systems and every time I want to push tokens I have to log out of github and log back in looking for the access key and all the information I need
it would be fantastic to be able to stay logged in to all 4 at the same time and switch between one and the other only at the time of the push

**Comment by PavelLaptev** [comment]
Hi @kkratterf thank you for your feedback. That's an interesting idea, I'll try to add next release.

**Comment by kkratterf** [comment]
Thanks 🫶 and congratulations again for the plugin, super useful

**Comment by Milenoi** [comment]
Any updates on this? It would be really helpful. we use this plugin but its a pain to change its values when you work on multiple figma files for different clients!

**Comment by PavelLaptev** [comment]
@Milenoi yes. I was looking to close the issue. Hopefully can do it this weekend 🤞 Stay tuned

**Comment by Milenoi** [comment]
Wow!! This would be so perfect! Wish you a productive good weekend!

**Comment by PavelLaptev** [comment]
@Milenoi danke!

**Comment by PavelLaptev** [comment]
The multi-profile feature is now live.
Here is how it should work https://github.com/tokens-bruecke/figma-plugin#multiple-profiles

https://github.com/user-attachments/assets/2b2f09d4-5163-4bdf-ae54-6862ed36db4f

**Comment by Milenoi** [comment]
This is incredible. i checkd it out and its exactly what we wanted and needed. thank you so much for this great work and effort! Very appreciated!!
